markup/tree-folders-selectable.html

```diff
@@ -12,7 +12,7 @@
 			</button>
 		</div>
 		<ul class="tree-branch-children" role="group"></ul>
-		<div class="tree-loader" role="alert">Loading...</div>
+		<div class="tree-loader"><span role="alert">Loading...</span></div>
```

Comment:
You said "you can't have `role="alert"` on li tags" but there are also a lot of changes here on div tags too.  Are you fixing more than necessary here?